src/sql/workbench/services/connection/browser/connectionBrowseTab.ts

```diff
@@ -538,21 +555,67 @@ interface ITreeExplorerTemplateData {
 	container: HTMLElement;
 	resourceLabel: IResourceLabel;
 	icon: HTMLElement;
-	// actionBar: ActionBar;
+	actionBar: ActionBar;
+}
+
+class TreeMenus {
```

Comment:
This name seems a little too generic for something that's pretty specific to the browse tree and it's also not really a "menu" itself. Would suggest something else like `ConnectionBrowseTreeMenuProvider` or similar

Reply:
this is a local class and won't be exported, but sure, I can update the name